samples/framework/libraries/migrate-library-csproj/src/Car/Car.csproj

```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net45;net451;netstandard1.6</TargetFrameworks>
```

Comment:
I wanted to show that the existing solution, which targeted net45 could still be built. I can do net45;net46;netstandard1.6?